tests/Unit.Tests/Controllers/FilterControllerTests.cs

```diff
@@ -146,5 +148,32 @@ public void QualificationGetRedirectsToNewApp()
             _redirectUrlMock.Verify(x => x.RedirectToNewApp(), Times.AtLeastOnce);
             result.Url.Should().Be(actualRedirectPath);
         }
+
+        [Test]
+        public void SubjectWizardGetHttpGetTest()
+        {
+            var result = _filterController.SubjectWizardGet(_resultsFilter);
+            result.Should().BeOfType<ViewResult>();
+            var viewResult = result as ViewResult;
+            viewResult?.Model.Should().BeOfType(typeof(SubjectFilterViewModel));
+            _mockApi.Verify(x => x.GetSubjectAreas(), Times.AtMostOnce);
```

Comment:
why `AtMostOnce`? Surely you know how many times it would be called?

Reply:
fixed